Add a Security Advisories page to the website (#128)
apache/jena-site: source/about_jena/security-advisories.md

---
title: Jena Security Advisories
---

# Security Issue Policy

> ## Process

Reviewer: The title becomes an H1 on the page. So that's two H1 then an H2.

---
![image](https://user-images.githubusercontent.com/101362/202807156-428892d1-c5a3-4929-95de-f38a6b3a7ed4.png)

---

For appearance, drop the "# " ; adds some text to separate the H1 from the H2 Process.

Author: Have added a brief into paragraph

Jena follows the standard [ASF Security for Committers](https://www.apache.org/security/committers.html) policy for
reporting and addressing security issues.

If you think you have identified a Security issue in our project please refer to that policy for how to report it, and
the process that the Jena Project Management Committee (PMC) will follow in addressing the issue.

## Single Supported Version

> As a project with a relatively small developer community Apache Jena only has the resources to maintain a single release

Reviewer: ```suggestion
As a project, Apache Jena only has the resources to maintain a single release
```
No need to apologise!

Author: Suggestions applied

> version.  Therefore any accepted security issue reported will be fixed by developing a fix for our `main` branch.  We
> will then make a release with the fix in a timeframe appropriate to the severity of the issue.  

Reviewer: ```suggestion
version.  Any accepted security issue will be fixed in a future release in a timeframe appropriate to the severity of the issue.  
```
We're suppose to not highlight security code fixes until the CVE is public (e.g. no git commit reference). Being git, we can't have private branches built no need to highlight the development.

2](../documentation/tdb2/) or other 3rd party vendor alternatives.

Apache Jena would like to thank Crilwa & LaNyer640 for reporting this issue

## CVE-2022-28890 - Processing External DTDs

[CVE-2022-28890](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-28890) affects the RDF/XML parser in Jena 4.4.0
only

Users should upgrade to latest Jena 4.x [release](../download/) available.

Apache Jena would like to thank Feras Daragma, Avishag Shapira & Amit Laish (GE Digital, Cyber Security Lab) for their
report.

## CVE-2021-39239 - XML External Entity (XXE) Vulnerability

[CVE-2021-39239](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-39239) affects XML parsing up to Jena 4.1.0

Users should upgrade to latest Jena 4.x [release](../download/) available.

## CVE-2021-33192 - Display information UI XSS in Apache Jena Fuseki

[CVE-2021-33192](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-33192) affected
[Fuseki](../documentation/fuseki2/) 2.0.0 through 4.0.0

Users should upgrade to latest Jena 4.x [release](../download/) available.

# CVEs in Jena Dependencies

> The following advisories are CVE's in Jena's dependencies that may affect users of Jena, as with Jena specific CVEs our

Reviewer 2: CVEs (for consistency? 😬)

Author: Fixed
